course_catalog/management/commands/backpopulate_youtube_data.py

```diff
@@ -57,3 +85,25 @@ def handle(self, *args, **options):
             self.stdout.write(
                 f"Fetched {result} YouTube videos in {total_seconds} seconds"
             )
+        elif command == "transcripts":
+            video_count = get_youtube_videos_for_transcripts_job(
```

Comment:
This pulls and filters the videos twice, which is not great but I think it's important to display the video count before the job starts since the job can take a few hours to run.